[Feat] 바텀 시트 개발

soohyn opened the pull request:
## PR 유형
- [X] 새로운 기능 추가
- [] 버그 수정
- [] CSS 등 사용자 UI 디자인 변경
- [] 코드에 영향을 주지 않는 변경사항(오타 수정, 탭 사이즈 변경, 변수명 변경)
- [] 코드 리팩토링
- [] 주석 추가 및 수정
- [] 문서 수정
- [] 테스트 추가, 테스트 리팩토링
- [] 빌드 부분 혹은 패키지 매니저 수정
- [] 파일 혹은 폴더명 수정
- [] 파일 혹은 폴더 삭제

## PR 체크리스트
<!-- PR이 다음 요구 사항을 충족하는지 확인하세요. -->
- [X] 커밋 메시지 컨벤션에 맞게 작성했습니다.
- [X] 변경 사항에 대한 테스트를 했습니다.(버그 수정/기능에 대한 테스트).

### PR 상세
- 바텀 시트 개발
- 애니메이션 제외 개발 완료

## 이슈
resolves # 25

soohyn committed:
[Design] navbar 하단 고정

- absolute

Resolves 바텀 시트 개발 #25

soohyn committed:
[Design] navbar background color

- bg-white 추가

Resolves 바텀 시트 개발 #25

soohyn committed:
[Design] radius 값 변경

- radius md 값 1rem으로변경
- radius lg 값 추가 1.25rem

Resolves 바텀 시트 개발 #25

soohyn committed:
[Feat] Divider 개발

- 오직 bottom sheet에서의 사용만을 생각하고 만들었음요

Resolves 바텀 시트 개발 #25

soohyn committed:
[Design] bottomSheet css 파일 생성

- 스크롤바 숨기는 css class 작성

Resolves 바텀 시트 개발 #25

soohyn committed:
[Feat] BottomSheet 컴포넌트 생성

- snap 기능 제외하고 완료

Resolves 바텀 시트 개발 #25

soohyn committed:
[Style] BottomSheet 위치 수정

- 부모 컴포넌트에서 relative 지워서 layout 참조하게 바꿈

Resolves 바텀 시트 개발 #25

soohyn committed:
Merge branch 'develop' of https://github.com/hyny-hyuny/gabom-react into feature/25-bottomSheet

juhyuny commented on src/styles/components/bottomSheet.css:
tailwind.css에 공통 스타일로 추가해 둔 거 있습니당!

juhyuny commented on src/styles/main.css:
overflow: hidden; 삭제 논의 완료!

soohyn committed:
[BugFix] navbar position fixed로 수정

- layout css에 테스트 코드인 overflow hidden 으로 인한 오류
- navbar fixed로 수정
- layout 하단에 여백 추가

Resolves 바텀 시트 개발 #25

soohyn committed:
[Refact] 공통 스크롤바 숨기기 css 사용

- bottom sheet css 제거

Resolves 바텀 시트 개발 #25

juhyuny reviewed (approved):
확인했습니당!
